js/main.js

```diff
@@ -168,6 +168,15 @@ function generateStorage(target) {
 }
 
 function handleHeart(target) {
+  function updateHearIcon(newImgURL) {
```

Comment:
`updateHeartIcon`, this says `updateHearIcon`. Also, does this need to be inside of the `handleHeart` function definition? If possible, it should be defined in the global scope with the rest of your functions.